boards/BMS/Inc/Io/Io_Airs.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <stdbool.h>
+
+/**
+ * Check if AIR+ is on
+ * @return true if AIR+ is on. Else, false
+ */
+bool Io_Airs_IsAirPositiveOn(void);
```

Comment:
I'd put the AIR- first so it's consistent with the App code

Reply:
Sounds good. I applied the changes